app/src/main/resources/logback.xml

```diff
@@ -1,14 +1,41 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-    <property name="defaultPattern" value="%-63(%yellow(%date{ISO8601}) %green([%.13thread])) %highlight(%-5level) %-50(%cyan(%logger{20})) - %msg%n"/>
+    <!-- log directory -->
+    <property name="LOG_HOME" value="logs"/>
+
+    <!-- Console log pattern with funky colours -->
+    <property name="consolePattern"
+              value="%-63(%yellow(%date{ISO8601}) %green([%.13thread])) %highlight(%-5level) %-50(%cyan(%logger{20})) - %msg%n"/>
+
+    <!-- File log pattern without colours -->
+    <property name="filePattern"
+              value="%-63(%date{ISO8601} [%.13thread]) %-5level %-50(%logger{20}) - %msg%n"/>
+
+    <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>${LOG_HOME}/explorer.log</file>
+
+        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+            <level>DEBUG</level>
+        </filter>
+
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+            <fileNamePattern>${LOG_HOME}/archived/explorer.%d{yyyy-MM-dd}.%i.log.gz</fileNamePattern>
+            <!-- max log file size before starting a new log file -->
+            <maxFileSize>10MB</maxFileSize>
```

Comment:
Let's add a totalSizeCap like here: https://github.com/alephium/alephium/blob/master/flow/src/main/resources/logback.xml#L54-L59

Reply:
Maybe we could use the exact same log file as full node, so we could reuse them and improve them in the same way

Reply:
:+1: but I agree with Simer, for log file we should deactivate colors.

Reply:
Agree. Let's remove for full node too.